src/library.js

```diff
@@ -5214,6 +5214,24 @@ LibraryManager.library = {
     return thisProgram || './this.program';
 #endif
   },
+
+  $autoResumeAudioContext: function(ctx) {
+    ['keydown', 'mousedown'].forEach(function(event) {
+      // Browsers will only allow audio to play after a user interaction. Listen
+      // for a mouse click on the document and on a canvas, if one exists.
+      document.addEventListener(event, function resumeAudioOnDocument() {
+        if (ctx.state === 'suspended') ctx.resume();
+        document.removeEventListener(event, resumeAudioOnDocument);
```

Comment:
Using the `{once: true}` option here would be nice. (`once` does not work on IE, but IE does not have Web Audio API either)

Reply:
Sadly it looks like Web Audio was present in many releases of Chrome and Firefox before `once` was added. So we still need to ifdef here. But that seems worth it, I added a helper function now with that applied.